mmv1/api/resource.go

```diff
@@ -1830,3 +1833,36 @@ func (r Resource) ShouldGenerateSweepers() bool {
 	}
 	return true
 }
+
+func (r Resource) GithubURL() string {
+	if r.githubUrl != "" {
+		return r.githubUrl
+	}
+
+	r.githubUrl = GITHUB_BASE_URL + r.SourceYamlFile
+	return r.githubUrl
+}
+
+func (r Resource) CodeHeader(templatePath string) string {
+	templateUrl := GITHUB_BASE_URL + templatePath
+
+	return fmt.Sprintf(`// ----------------------------------------------------------------------------
+//
+//     ***     AUTO GENERATED CODE    ***    Type: MMv1     ***
+//
+// ----------------------------------------------------------------------------
+//
+//     This code is generated by Magic Modules using the following:
+// 
```

Comment:
I suspect the trailing whitespace at the end of the line is the linter issue. Can you try to remove it?

Reply:
yeah you're right !